inference-engine/src/vpu/myriad_plugin/myriad_config.cpp

```diff
@@ -16,13 +16,7 @@ namespace vpu {
 namespace MyriadPlugin {
 
 const std::unordered_set<std::string>& MyriadConfig::getCompileOptions() const {
-IE_SUPPRESS_DEPRECATED_START
-    static const std::unordered_set<std::string> options = merge(ParsedConfig::getCompileOptions(), {
-        VPU_MYRIAD_CONFIG_KEY(PLATFORM),
-    });
-IE_SUPPRESS_DEPRECATED_END
-
-    return options;
+    return ParsedConfig::getCompileOptions();
```

Comment:
small nitpick: if ParsedConfig is a parent class then we do not need an overridden method in MyriadConfig class